GENERAL:
  timerPeriod:        0.1           # period of path planning task in [s]

LOCAL:
  collisionRange:     6             # cones in this range will be checked for collision - in [m]
  goalBias:           0.1   
  goalRadius:         1             # acceptance range in [m]
  iterations:         500           # iterations for planning
  maxConeDist:        6             # maximal distance between consecutive cones in [m]
  maxNumOfNodes:      2000          # limits number of nodes in searchtree
  maxVelocity:        10            # in [m/s]
  minCost:            0.0           # cost threshold used to filter nodes when tracing back the path to root
  minDeviation:       0.01          # minimum difference between states
  resolution:         0.05          # interpolation steps in [m]
  rewireRange:        1             # Range in which connection is checked in [m]
  sampleRange:        3             # used for random sampling near the detected cones, range in [m]
  simulationTimeStep: 0.1           # in [s]
  thetaWeight:        50           # weight of orientation difference in distance calculation

GLOBAL:
  collisionRange:     6             # cones in this range will be checked for collision - in [m]
  goalBias:           0.1   
  goalRadius:         0.5           # acceptance range in [m]
  iterations:         50            # iterations for planning
  maxConeDist:        6             # maximal distance between consecutive cones in [m]
  maxNumOfNodes:      9000          # limits number of nodes in searchtree
  maxVelocity:        10            # in [m/s]
  minCost:            10.0          # cost threshold used to filter nodes when tracing back the path to root
  minDeviation:       0.01          # minimum difference between states
  resolution:         0.05          # interpolation steps in [m]
  rewireRange:        1             # Range in which connection is checked in [m]
  sampleRange:        3             # used for random sampling near the detected cones, range in [m]
  simulationTimeStep: 0.1           # in [s]
  thetaWeight:        50           # weight of orientation difference in distance calculation

VEHICLE:
  track:              1.2           # in [m]
  wheelBase:          1.54          # in [m]
  simType:            "HOLONOMIC_CONSTRAINED"   # can be "HOLONOMIC" , "HOLONOMIC_CONSTRAINED" , "BICYCLE_SIMPLE" , "BICYCLE"
  costType:           "DISTANCE"    # can be "DISTANCE" , "TIME"

MAP:
  goalHorizon:        15            # maximal distance for local goal state in [m]
